taxi/views.py

```diff
@@ -28,6 +30,20 @@
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def assign_driver_to_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    car.drivers.add(request.user)
+    return redirect(reverse("taxi:car-detail", args=[pk]))
+
+
+@login_required
+def unassign_driver_from_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    car.drivers.remove(request.user)
```

Comment:
Similar to the `assign_driver_to_car` function, ensure that `request.user` is a `Driver` instance before attempting to remove it from the car's drivers.